crates/network-scanner/src/mdns.rs

```diff
@@ -23,94 +23,133 @@ impl MdnsDaemon {
     }
 }
 
+const SERVICES_INTRESTED: [Protocol; 11] = [
```

Comment:
typo: "intrested". I guess this is the list of protocols we are interested in?

naming: Is "service" the right word for this? Why not "protocol"? For example **RDP** is the name of the protocol and both **Remote Desktop Services (RDS)** and **XRDP** are services for this same protocol. According to the mDNS specification `_googlecast._tcp.local` would be a service name too.
Good alternative: "[service type](http://www.dns-sd.org/servicetypes.html)".

Reply:
updated to ServiceType